package com.jimmy.portal.job.dto;

import com.jimmy.portal.company.dto.JobDto;

import java.time.Instant;

public record JobApplicationDto(
        Long id,
        Long userId,
        String userName,
        String userEmail,
        String userMobileNumber,
        ProfileDto userProfile,
        JobDto job,
        Instant appliedAt,
        String status,
        String coverLetter,
        String notes
) {
}
